modules/serverless-ossm-setup-with-kourier.adoc

```diff
@@ -72,13 +60,28 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          serving.knative.openshift.io/system-namespace: "true"
+          knative.openshift.io/part-of: "openshift-serverless"
   podSelector: {}
   policyTypes:
   - Ingress
 ...
 ----
 <1> Add the namespace that you want to integrate with {SMProductShortName}.
++
+[NOTE]
```

Comment:
This note could be part of an additional step, given that it includes two procedural commands. For example, you could change it to a step with something like the following format:

> b.  If you are using OpenShift Serverless 1.21 or earlier, add the `knative.openshift.io/part-of` label to the `knative-serving` and `knative-serving-ingress` namespaces:
> 
> i. Add the label to the `knative-serving` namespace:
> 
> $ oc label namespace knative-serving knative.openshift.io/part-of=openshift-serverless
> 
> ii. Add the label to the `knative-serving-ingress` namespace:
> 
> $ $ oc label namespace knative-serving-ingress knative.openshift.io/part-of=openshift-serverless
> 
> **Note**: The knative.openshift.io/part-of: "openshift-serverless" label was added in OpenShift Serverless 1.22.0.

In the example step, I have also added substeps prior to each code block, to provide the reader with a clear understanding about what each command does explicitly.